### PR TITLE
Make lapwing-aio work

### PR DESCRIPTION
**Pull request by tocklime**
This is a set of overrides that make lapwing-aio work. It's mostly dependent on other plugins. I think what I've got isn't awful. It works, at least.

### CODE REVIEW

**Comment by tocklime** [comment]
~~actually, it might be that the submodules aren't working right~~

at least repeat-last-translation is working, so I guess the others are too...

**Comment by dnaq** [comment]
Thanks!
Can you change the `super` references to `self` references? Other than that lgtm

File: overrides.nix [hunk 1]
```diff
@@ -16,4 +16,13 @@ self: super: {
   plover_dict_commands = super.plover_dict_commands.overrideAttrs (old: {
     propagatedBuildInputs = [ setuptools-scm ];
   });
+  plover_lapwing_aio = super.plover_lapwing_aio.overrideAttrs (old: {
+    propagatedBuildInputs = [
+      super.plover_stitching
```
Comment (dnaq) [inline overrides.nix]: self

File: overrides.nix [hunk 2]
```diff
@@ -16,4 +16,13 @@ self: super: {
...
+      super.plover_python_dictionary
```
Comment (dnaq) [inline overrides.nix]: self

File: overrides.nix [hunk 3]
```diff
@@ -16,4 +16,13 @@ self: super: {
...
+      super.plover-modal-dictionary
```
Comment (dnaq) [inline overrides.nix]: self

File: overrides.nix [hunk 4]
```diff
@@ -16,4 +16,13 @@ self: super: {
...
+      super.plover_last_translation
```
Comment (dnaq) [inline overrides.nix]: self

**Comment by tocklime** [comment]
Done `s/super/self/`.

**Pull request merged**